bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -358,6 +360,14 @@ impl Script {
         }
     }
 
+    /// Check if this is an OP_RETURN that obeys Bitcoin Core standardness policy. What this
+    /// function considers to be standard may change without warning pending Bitcoin Core
+    /// changes.
```

Comment:
I've recently realized that maybe we should have some better way of handling Core versions. I'm not really sure how. I was thinking of having an enum `CoreVersion` but that'd force us into supporting old ones forever. I was also thinking of having a special crate that tracks the versions (or maybe as a part of jsonrpc?) but I'm not sure if it'll work well with `Network`.

Anyway, this is acceptable since it's what we do already.